### ISSUE TITLE
Date/Time Field to populate posts published date/time value

### ISSUE THREAD

**Issue by zushiba** [pods-framework/pods#3973]
I'd like to request an option for the Date/Time field to populate the Date/Time for the post. I do realize that it's editable via the post edit screen but that's confusing to my users who still call the computer the "Brain". 

It would also make the plugin compatible with other plugins that are capable of accessing custom post types and use the date for certain functions such as [W_pFullcalendar](https://wordpress.org/plugins/wp-fullcalendar/) which uses the the posts publish date/time field.

**Comment by jimtrue** [comment]
@zushiba Would you consider this an option like 'Link this Date/Time Field to Post Date?' option or something of that nature?

**Comment by zushiba** [comment]
Yes that would be fantastic, it would make things easier for my users to understand.
